www/js/app.js

```diff
@@ -166,7 +165,18 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     
     // Unique identifier of the article expected to be displayed
     var expectedArticleURLToBeDisplayed = "";
-    
+ 
+    // define variable for dark preference for matchMedia
+    var darkPreference = window.matchMedia('(prefers-color-scheme:dark)');
+    if (!window.matchMedia) document.getElementById('appThemeSelect').options[0].style.display = "none";
+    if (!window.matchMedia) document.getElementById('appThemeSelect').options[1].style.display = "none";
```

Comment:
Again here, just use a single `if` condition, and inside it set the two options to hidden. In general, if you find you are doing the same test twice, you are probably writing inefficient code!

Reply:
@Jaifroid, `document.getElementById('appThemeSelect').options[0].style.display = "none";` 
Should I write these expressions as it is or use the `for` loop.

Reply:
I think `for` is overkill here. If there were several options, yes, but just two it's OK just to set them both in the `if` block.

Reply:
Ohkay sir